config/sizes.json

```diff
@@ -34,5 +34,9 @@
     "handlebars-intl/dist/handlebars-intl.min.js": {
         "bytes": 7267,
         "kbs": 7.1
+    },
+    "ember-intl/dist/ember-intl.min.js": {
+        "bytes": 21362,
+        "kbs": 25
```

Comment:
I'm having trouble figuring out a method of calculating this, since the addon is not disted as a standalone js asset.  

This is really the size of the "shimmed" dist which has some added weight (contains a closed of copy of intl-relativeformat and intl-messageformat) - it's also not minified.  

I was thinking of doing the "minification" within the formatjs brocfile.  Thoughts?